defaults/main.yml

```diff
@@ -27,6 +33,7 @@ consul_install_remotely: false
 consul_bin_path: "/usr/local/bin"
 consul_config_path: "/etc/consul"
 consul_configd_path: "/etc/consul.d"
+consul_bootstrap_state: "/etc/consul/.consul_bootstrapped"
```

Comment:
Bootstrap state is another path I needed to change for Windows, and consolidating it with the other paths seemed to be the smart choice.